src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGainPID.java

```diff
@@ -0,0 +1,14 @@
+package org.usfirst.frc4904.robot.humaninterface.drivers;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.custom.MCChassisController;
+
+public class NathanGainPID extends MCChassisController {
+	public static final double MAXIMUM_DEGREES_PER_SECOND = 100; // TODO
```

Comment:
If this is ever going to be used in other human interface classes, it should probably go in the human interface config. If not, in here is fine.